mysqld_exporter.go

```diff
@@ -46,6 +46,14 @@ var (
 		"collect.perf_schema.indexiowaitstime", false,
 		"Collect time metrics from performance_schema.table_io_waits_summary_by_index_usage",
 	)
+	perfEventsStatements = flag.Bool(
+		"collect.perf_schema.eventsstatements", false,
```

Comment:
Yes, the table name (see the next line) is `performance_schema.events_statements_history`.  :-/